postfix/map.jinja

```diff
@@ -16,6 +16,10 @@
             '/etc/hosts', '/etc/localtime',
             '/etc/ssl/certs/ca-certificates.crt',
         ],
+        'ssl': {
+          'cert':  'AAAAA CERT AAAAA',
+          'chain': 'AAAA  CHAIN AAAA',
```

Comment:
Don't add placeholders like this. Better ensure that proper `server.ssl` structure exists in code that needs it.